.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: Test
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
```

Comment:
This will skip running the test suite if the only files changes are top level markdown files like the README.